src/cli/configuration_builder.ts

```diff
@@ -157,15 +163,43 @@ export default class ConfigurationBuilder {
     return _.uniq(featureDirs)
   }
 
+  isPublishing(): boolean {
+    return (
+      this.options.publish ||
+      this.isTruthyString(process.env.CUCUMBER_PUBLISH_ENABLED) ||
+      process.env.CUCUMBER_PUBLISH_TOKEN !== undefined
+    )
+  }
+
+  isPublishAdvertisementSuppressed(): boolean {
+    return (
+      this.options.publishQuiet ||
+      this.isTruthyString(process.env.CUCUMBER_PUBLISH_QUIET)
+    )
+  }
+
   getFormats(): IConfigurationFormat[] {
     const mapping: { [key: string]: string } = { '': 'progress' }
     this.options.format.forEach((format) => {
       const [type, outputTo] = OptionSplitter.split(format)
       mapping[outputTo] = type
     })
+    if (this.isPublishing()) {
+      const publishUrl =
+        // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+        process.env.CUCUMBER_PUBLISH_URL || DEFAULT_CUCUMBER_PUBLISH_URL
```

Comment:
For this, we also have a helper is src/value_checker.

`valueOrDefault(a, b)` which returns a if not null / undefined, otherwise b

Using that should be able to remove the eslint disable

Reply:
Not sure but this newer syntax might work too:

```typescript
process.env.CUCUMBER_PUBLISH_URL ?? DEFAULT_CUCUMBER_PUBLISH_URL
```

Reply:
resolved by 860f430b6bc5235e51ef4dc51318556f68ef550c